src/containers/Storage/Pdisk/Pdisk.tsx

```diff
@@ -8,38 +8,37 @@ import {InternalLink} from '../../../components/InternalLink';
 
 import routes, {createHref} from '../../../routes';
 import type {RequiredField} from '../../../types';
+import {EFlag} from '../../../types/api/enums';
 import {TPDiskStateInfo, TPDiskState} from '../../../types/api/pdisk';
 import {getPDiskId} from '../../../utils';
 import {getPDiskType} from '../../../utils/pdisk';
 import {bytesToGB} from '../../../utils/utils';
 
 import {STRUCTURE} from '../../Node/NodePages';
 
-import DiskStateProgressBar, {
-    diskProgressColors,
-} from '../DiskStateProgressBar/DiskStateProgressBar';
+import {DiskStateProgressBar, EDiskStateSeverity} from '../DiskStateProgressBar';
 
-import {colorSeverity, NOT_AVAILABLE_SEVERITY} from '../utils';
+import {NOT_AVAILABLE_SEVERITY} from '../utils';
 
 import './Pdisk.scss';
 
 const b = cn('pdisk-storage');
 
 const stateSeverity = {
-    [TPDiskState.Initial]: 0,
-    [TPDiskState.Normal]: 1,
-    [TPDiskState.InitialFormatRead]: 3,
-    [TPDiskState.InitialSysLogRead]: 3,
-    [TPDiskState.InitialCommonLogRead]: 3,
-    [TPDiskState.InitialFormatReadError]: 5,
-    [TPDiskState.InitialSysLogReadError]: 5,
-    [TPDiskState.InitialSysLogParseError]: 5,
-    [TPDiskState.InitialCommonLogReadError]: 5,
-    [TPDiskState.InitialCommonLogParseError]: 5,
-    [TPDiskState.CommonLoggerInitError]: 5,
-    [TPDiskState.OpenFileError]: 5,
-    [TPDiskState.ChunkQuotaError]: 5,
-    [TPDiskState.DeviceIoError]: 5,
+    [TPDiskState.Initial]: EDiskStateSeverity.Grey,
```

Comment:
those are direct references to the possible component prop values, should use enum